ModularTegustation/tegu_traitor_code/uplink_items_tegu.dm

```diff
@@ -7,6 +7,24 @@
 	restricted_roles = list("Curator")
 	limited_stock = 1 //for testing at least
 
+/datum/uplink_item/role_restricted/hypnotic_flash/psych
+	name = "Advanced Hypnotic Flash"
+	desc = "A modified flash able to hypnotize targets. This model is capable of beaming orders directly into target's mind, instantly. \
+			The only downside is that it requires about 30 seconds to recharge. Still doesn't work against mental barriers, such as mindshield implants."
+	item = /obj/item/assembly/flash/hypnotic/adv
+	cost = 10
+	cant_discount = FALSE
+	restricted_roles = list("Psychologist")
+	limited_stock = 1
+
+/datum/uplink_item/role_restricted/symptom_box
+	name = "Virology Symptoms Box"
+	desc = "Box full of advanced virology symptoms, extracted from Tiger Co. abandoned laborotories. \
```

Comment:
Should this really be Tiger Co? Interdyne Pharmaceutics seems more fitting.

Reply:
Damn /tg/ and their lore changes. I'll keep it Tiger Co, for these ones.
I was planning to add more symptoms later, so new ones can be Interdyne Pharma.